lib/middleware/repos-attach-metadata.js

```diff
@@ -0,0 +1,22 @@
+const { SlackWorkspace, SlackUser, GitHubUser } = require('../models');
```

Comment:
I think we don't need this middleware now that we have `requireGitHubUser` middleware: https://github.com/integrations/slack/pull/606/files#diff-f02746daec04fc7c4e865ad953a4b5d2R1

Reply:
True